src/Estimators/include/BipedalLocomotion/Estimators/FloatingBaseEstimator.h

```diff
@@ -0,0 +1,268 @@
+
+        /**
+        * Get relative pose between IMU and the feet
+        * @param[in] encoders joint positions through encoder measurements
+        * @param[out] IMU_H_l_foot pose of the left foot contact frame with respect to the IMU frame
+        * @param[out] IMU_H_r_foot pose of the right foot contact frame with respect to the IMU frame
+        * @return True in case of success, false otherwise.
+        */
+        bool getIMU_H_feet(const iDynTree::JointPosDoubleArray& encoders,
+                           iDynTree::Transform& IMU_H_l_foot,
+                           iDynTree::Transform& IMU_H_r_foot);
+
+        /**
+        * Get the base link pose and velocity from the estimated IMU pose and velocity
+        * @note the input and output velocities are both specified in mixed-velocity representation
+        * @param[in] A_H_IMU pose of the IMU in the world
+        * @param[in] v_IMU mixed-trivialized velocity of the IMU in the world
+        * @param[out] A_H_B pose of the base link in the world
+        * @param[out] v_B mixed-trivialized velocity of the base link in the world
+        * @return True in case of success, false otherwise.
+        */
+        bool getBaseStateFromIMUState(const iDynTree::Transform& A_H_IMU, const iDynTree::Twist& v_IMU,
+                                      iDynTree::Transform& A_H_B, iDynTree::Twist& v_B);
+
+        /**
+         * Getters
+         */
+        const int& nrJoints() const { return m_nr_joints; }
+        const std::string& baseLink() const { return m_base_link; }
+        const std::string& baseLinkIMU() const { return m_base_imu_frame; }
+        const std::string& leftFootContactFrame() const { return m_l_foot_contact_frame; }
+        const std::string& rightFootContactFrame() const { return m_r_foot_contact_frame; }
+        const iDynTree::Transform& base_H_IMU() const { return m_base_H_imu; }
+        const bool& isModelSet() const { return m_model_set; }
+
+    private:
+        std::string m_base_link{""}; /**< name of the floating base link*/
+        std::string m_base_imu_frame{""}; /**< name of the IMU frame rigidly attached to the floating base link*/
+        std::string m_l_foot_contact_frame{""}; /**< name of the left foot contact frame expected to be in contact with the environment*/
+        std::string m_r_foot_contact_frame{""}; /**< name of the right foot contact frame expected to be in contact with the environment*/
+        iDynTree::FrameIndex m_base_link_idx{iDynTree::FRAME_INVALID_INDEX}; /**< base link's frame index in the loaded model*/
+        iDynTree::FrameIndex m_base_imu_idx{iDynTree::FRAME_INVALID_INDEX}; /**< IMU index in the loaded model*/
+        iDynTree::FrameIndex m_l_foot_contact_idx{iDynTree::FRAME_INVALID_INDEX}; /**< Left foot contact frame index in the loaded model*/
+        iDynTree::FrameIndex m_r_foot_contact_idx{iDynTree::FRAME_INVALID_INDEX}; /**< Right foot contact freame index in the loaded model*/
```

Comment:
actually we use also camel case also for variables (i.e. `m_l_foot_contact_frame` -> `m_lFootContactFrame` ).

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/79/commits/f46bd5b271b6e7ea77da83c41b90178157aac069 :D